clean_code_main/clean_code_checks/y_check_method_output_param.clas.abap

```diff
@@ -127,17 +131,20 @@ CLASS Y_CHECK_METHOD_OUTPUT_PARAM IMPLEMENTATION.
 
     ENDLOOP.
 
-    DATA(check_configuration) = detect_check_configuration( threshold = 1
-                                                            include = get_include( p_level = statement-level ) ).
-    IF check_configuration IS INITIAL.
+    IF has_error( ) = abap_false.
       RETURN.
     ENDIF.
 
-    IF calculate_param_combination( ) > check_configuration-threshold.
-      raise_error( statement_level     = statement-level
-                   statement_index     = index
-                   statement_from      = statement-from
-                   error_priority      = check_configuration-prio ).
+    DATA(check_configuration) = detect_check_configuration( statement ).
```

Comment:
Shouldn't we set the default threshold to 0 as it is being done in the check_*_usage checks?

Reply:
the check did not use a threshold at all.